integration/test/crmapping/testdata/simple_azureconfig.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: provider.giantswarm.io/v1alpha1
+kind: AzureConfig
+metadata:
+  creationTimestamp: "2020-08-10T11:03:14Z"
+  finalizers:
+  - operatorkit.giantswarm.io/azure-operator
+  - operatorkit.giantswarm.io/legacy-controller
+  generation: 1
+  labels:
+    azure-operator.giantswarm.io/version: 4.2.0
+    giantswarm.io/cluster: c6fme
+    release.giantswarm.io/version: 12.0.0
+  name: c6fme
+  namespace: default
+  resourceVersion: "176010431"
+  selfLink: /apis/provider.giantswarm.io/v1alpha1/namespaces/default/azureconfigs/c6fme
+  uid: d8fcb3cf-4561-4120-91c2-43e3633d5df2
+spec:
+  azure:
+    availabilityZones:
+    - 1
+    credentialSecret:
+      name: credential-default
+      namespace: giantswarm
+    dnsZones:
+      api:
+        name: ghost.westeurope.azure.gigantic.io
+        resourceGroup: ghost
+      etcd:
+        name: ghost.westeurope.azure.gigantic.io
+        resourceGroup: ghost
+      ingress:
+        name: ghost.westeurope.azure.gigantic.io
+        resourceGroup: ghost
+    masters:
+    - dockerVolumeSizeGB: 50
+      kubeletVolumeSizeGB: 100
+      vmSize: Standard_D4s_v3
+    virtualNetwork:
+      calicoSubnetCIDR: 10.10.128.0/17
+      cidr: 10.10.0.0/16
+      masterSubnetCIDR: 10.10.0.0/24
+      workerSubnetCIDR: 10.10.1.0/24
+    workers:
+    - dockerVolumeSizeGB: 50
+      kubeletVolumeSizeGB: 100
+      vmSize: Standard_A2_v2
+  cluster:
+    calico:
+      cidr: 16
+      mtu: 1430
+      subnet: 10.1.0.0/16
+    customer:
+      id: giantswarm
+    docker:
+      daemon:
+        cidr: 172.17.0.1/16
+    etcd:
+      altNames: ""
+      domain: etcd.c6fme.k8s.ghost.westeurope.azure.gigantic.io
+      port: 2379
+      prefix: giantswarm.io
+    id: c6fme
+    kubernetes:
+      api:
+        clusterIPRange: 172.31.0.0/16
+        domain: api.c6fme.k8s.ghost.westeurope.azure.gigantic.io
+        securePort: 443
+      cloudProvider: azure
+      dns:
+        ip: 172.31.0.10
+      domain: cluster.local
+      ingressController:
+        docker:
+          image: quay.io/giantswarm/nginx-ingress-controller:0.9.0-beta.11
+        domain: ingress.c6fme.k8s.ghost.westeurope.azure.gigantic.io
+        insecurePort: 30010
+        securePort: 30011
+        wildcardDomain: '*.c6fme.k8s.ghost.westeurope.azure.gigantic.io'
+      kubelet:
+        altNames: kubernetes,kubernetes.default,kubernetes.default.svc,kubernetes.default.svc.cluster.local
+        domain: worker.c6fme.k8s.ghost.westeurope.azure.gigantic.io
+        labels: azure-operator.giantswarm.io/version=4.2.0,giantswarm.io/provider=azure
+        port: 10250
+      networkSetup:
+        docker:
+          image: quay.io/giantswarm/k8s-setup-network-environment:1f4ffc52095ac368847ce3428ea99b257003d9b9
+        kubeProxy:
+          conntrackMaxPerCore: 0
+      ssh:
+        userList:
+        - name: foobar
+          publicKey: ssh-rsa foobarbaz== foobar
```

Comment:
This CR is from real live cluster but I redacted SSH keys out (even though they are public keys).